src/man/firejail.txt

```diff
@@ -2859,6 +2865,9 @@ $ firejail \-\-tree
 .br
   11970:netblue:transmission-gtk
 
+.br
+Note: Support for this command is controlled in firejail.config with the
++\fBtracelog\fR option.
```

Comment:
```suggestion
  11970:netblue:transmission-gtk
.br

.br
Note: Support for this command is controlled in firejail.config with the
\fBtracelog\fR option.
```

Remove the extraneous `+` at the beginning and use the double `.br` commands to
ensure that there is a blank newline in between.  Example:

```
Line 1.
.br

.br
Line 2.
```

Note: There are better ways to accomplish this, but this is what is usually
done in this man page.

Lastly, the `tracelog` note should be moved back to the relevant section (it's
in the `--tree` section).